torch/onnx/_internal/diagnostics/infra/context.py

```diff
@@ -193,7 +197,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
-        return True
+        return None
```

Comment:
What does this change? Just for my learning

Reply:
Returning `True` will suppress the exception. `None` will allow it to propagate.

Reply:
Thanks!